bin/vue-build

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env node
+
+var fs = require('fs')
+var path = require('path')
+var program = require('commander')
+var chalk = require('chalk')
+var rm = require('rimraf').sync
+var webpack = require('webpack')
+var Server = require('webpack-dev-server')
+var webpackMerge = require('webpack-merge')
+var HtmlWebpackPlugin = require('html-webpack-plugin')
+var FriendlyErrorsPlugin = require('friendly-errors-webpack-plugin')
+var PostCompilePlugin = require('post-compile-webpack-plugin')
+var ProgressPlugin = require('webpack/lib/ProgressPlugin')
+var ExtractTextPlugin = require('extract-text-webpack-plugin')
+var isYarn = require('installed-by-yarn-globally')
+var loaders = require('../lib/loaders')
+var logger = require('../lib/logger')
+
+/**
+ * Usage.
+ */
+
+program
+  .usage('[entry]')
+  .option('--dist [directory]', 'Output directory for bundled files', 'dist')
+  .option('--port [port]', 'Server port', 4000)
+  .option('--host [host]', 'Server host', 'localhost')
+  .option('--prod, --production', 'Build for production', false)
+  .option('-c, --config [directory]', 'Use custom config directory')
+  .option('--no-config', 'You do not want global and local config')
+  .option('-o, --open', 'Open browser')
+  .parse(process.argv)
+
+var args = program.args
+var production = program.production
+
+process.env.NODE_ENV = production ? 'production' : 'development'
+
+// load config and webpack config from $config/.vue
+var configContext = typeof program.config === 'string' ? process.cwd() : require('user-home')
+var configDirectory = typeof program.config === 'string' ? program.config : '.vue'
+var localConfigPath = path.join(configContext, configDirectory, 'config.js')
+var localWebpackConfigPath = path.join(configContext, configDirectory, 'webpack.config.js')
+var localConfig
+var localWebpackConfig
+
+if (program.config !== false) {
+  var hasLocalConfig = fs.existsSync(localConfigPath)
+  var hasLocalWebpackConfig = fs.existsSync(localWebpackConfigPath)
+  if (hasLocalConfig) {
+    localConfig = require(localConfigPath)
+  }
+  if (hasLocalWebpackConfig) {
+    localWebpackConfig = require(localWebpackConfigPath)
+  }
+}
+
+var options = Object.assign({
+  entry: 'index.js'
```

Comment:
Because of this if you run only `vue build` you'll get `index.js does not exist, did you forget to create one?`. Maybe we should default to help output when no entry is provided?